e2e-tests/commands/shared-commands/assertions/assert-section-start-content.js

```diff
@@ -14,6 +15,7 @@ import { BUTTONS } from '../../../content-strings';
 const assertSectionStartContent = {
   intro: (expectedText) => cy.checkIntroText(expectedText),
   list: {
+    intro: (expectedText) => cy.checkText(listIntro(), expectedText),
```

Comment:
Instead of this, instances of `assertSectionStartContent.intro` could just use `cy.checkIntroText` instead?

Reply:
won't work as that checks data-cy of intro and these are list-intro

Reply:
I see sorry, misread.